block-storage-attacher/Dockerfile.builder

```diff
@@ -1,6 +1,6 @@
-FROM golang:1.15.2
+FROM golang:1.15.5
 
-ARG GOPROXY=off
+#ARG GOPROXY=off
```

Comment:
why is this removed?

Reply:
If we keep it on we are getting build errors